docs/src/walkthrough.md

```diff
@@ -87,46 +87,37 @@ To let us use `await` later, the rest of the file is wrapped in an `async` IIFE.
 })();
 ```
 
-### 1. Create `CRDTContainer`
+### 1. Create [`CRDTContainer`](./guide/containers.html)
 
-Our first real task is to get an instance of `CRDTContainer`. This is the entry point for a Collabs container; it connects your `Collab`s to the Collabs container host, which in turn connects them to other collaborators over the network.
+Our first real task is to get an instance of [`CRDTContainer`](./guide/containers.html). This is the entry point for a Collabs [container](./guide/containers.html); it connects your `Collab`s to the Collabs container host, which in turn connects them to other collaborators over the network.
```

Comment:
I think it would be better to link this to the CRDTContainer API docs (in @collabs/container), since the next sentence links to the Containers guide page.